### PR TITLE
Ccp 2409 add portals

### PR DESCRIPTION
**Pull request by Phmank**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

- [ ] Bug fix
- [X] New feature
- [ ] This change requires a documentation update

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I updated the CHANGELOG.md

### CODE REVIEW

File: extension-config.json [hunk 1]
```diff
@@ -20,6 +45,16 @@
         "path": "/v1/shop/%(shopId)s/sgc_extension_settings?parsed=true",
         "key": "value.@shopgate/youtube-pdp.videoProperty"
       }
+    },
+    "portalName": {
+      "type": "admin",
+      "destination": "frontend",
+      "default": "product.description.after",
+      "params": {
+        "type": "json",
```
Comment (Carsten89) [inline extension-config.json]:
```suggestion
        "type": "text",
```

File: frontend/helpers/getConfig.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { logger } from '@shopgate/pwa-core/helpers';
+
```
Comment (Carsten89) [inline frontend/helpers/getConfig.js]: please remove this file. its not needed. you can access the config directly without the helper fn here

File: frontend/helpers/getConfig.spec.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const mockedConfig = {};
+jest.mock('../config', () => mockedConfig, { virtual: true });
+
```
Comment (Carsten89) [inline frontend/helpers/getConfig.spec.js]: this file should also be removed.

**Review by Carsten89 — CHANGES_REQUESTED** [review]

File: extension-config.json (hunk 1 above)
Comment (Phmank) [reply extension-config.json]: Done

File: frontend/helpers/getConfig.js (hunk 2 above)
Comment (Phmank) [reply frontend/helpers/getConfig.js]: Done

File: frontend/helpers/getConfig.spec.js (hunk 3 above)
Comment (Phmank) [reply frontend/helpers/getConfig.spec.js]: Done

**Review by Carsten89 — APPROVED** [review]

**Pull request merged**